crates/primitives/src/genesis.rs

```diff
@@ -185,7 +185,15 @@ impl From<GenesisAccount> for Account {
     }
 }
 
-/// Represents a node's chain configuration.
+/// Defines core blockchain settings per block.
+///
+/// Stores essential network configurations persistently in the database.
+///
+/// Tailors unique settings for each network based on its genesis block.
+///
+/// Governs crucial blockchain behavior and adaptability.
+///
+/// Encapsulates parameters shaping network evolution and behavior.
 ///
```

Comment:
This is not really correct, this is not stored in the database. The previous comment is better IMO, but improving on that still would be valuable

Reply:
Fine, just removed the wrong part of the doc concerning database.